examples/slave/nuttx_arduino_due/export_nuttx_archive.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Something should be done to generalize this helper script at some point.

Reply:
For the record, we agreed it will only be used as a guidance example. So no will to make it generic.